pkg/service/client_jsonrpc.go

```diff
@@ -24,12 +24,9 @@ import (
 func forceNoChunkedEncoding(ctx context.Context, r *http.Request) context.Context {
 	r.TransferEncoding = []string{"identity"}
 
-	// read the body, set the content legth, and leave a new ReadCloser in Body
+	// read the body, set the content length, and leave a new ReadCloser in Body
 	bodyBuf := &bytes.Buffer{}
-	bodyReadBytes, err := io.Copy(bodyBuf, r.Body)
-	if err != nil {
-		panic(err)
-	}
+	bodyReadBytes, _ := io.Copy(bodyBuf, r.Body)
```

Comment:
I don't know if there's anything better we can do here. We don't have access to a logger and it's not trivial to get access.

Reply:
I agree ... what else can we do here 🤷

Reply:
It's worth a comment about why we're discarding it. But yeah 🤷

Reply:
Added a comment